### PR TITLE
Adds Ark Soft Instructor job for Mentors

### PR DESCRIPTION
**Pull request by ZomgPonies**
## What Does This PR Do
Based on https://github.com/ParadiseSS13/Paradise/pull/13398

Adds a new job, "Ark Soft Instructor", based on a similar job on CM: https://cm-ss13.com/wiki/Senior_Enlisted_Adviser
This is a mentor/admin exclusive job that lets mentors teach new players ICly.
They are not included in job prefs, but you can latejoin as one if you're a mentor/admin.
Their radio has all channels, similar to ArkRep.
Their access is based on ArkRep, except that they do NOT get access to personal lockers, crematorium, vault, keycard auth, gateway, ArkRep office. So their access is between that of an IAA and the ArkRep.

## Why It's Good For The Game
Allows mentors to teach new players ICly encouraging more roleplay in general.

## Changelog
:cl:
add: Added Ark Soft Instructor, a mentor/admin exclusive job aimed at teaching new players basic game mechanics.
fix: Adds bouncer spawn to Cyberiad for consistency.
/:cl:

### CODE REVIEW

**Comment by trololiver112** [comment]
LGTM!

File: code/game/jobs/job/support.dm [hunk 1]
```diff
@@ -521,6 +521,62 @@
...
+	outfit = /datum/outfit/job/arksoftinstructor
+
+/datum/outfit/job/arksoftinstructor
+	name = "Ark Soft Instructor"
+	jobtype = /datum/job/arksoftinstructor
+
+	uniform = /obj/item/clothing/under/rank/centcom/representative // formal Arksoft uniform
+	shoes = /obj/item/clothing/shoes/centcom // fancy shoes
+	head = /obj/item/clothing/head/beret/blue // generic blue beret
+	l_ear = /obj/item/radio/headset/heads/arkinstructor //
```
Comment (farie82) [inline code/game/jobs/job/support.dm:556]: empty comment here

File: code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -248,6 +248,13 @@
 	item_state = "headset"
 	ks2type = /obj/item/encryptionkey/heads/arkrep
 
+/obj/item/radio/headset/heads/arkinstructor
+	name = "ark soft instructor's headset"
```
Comment (farie82) [inline code/game/objects/items/devices/radio/headset.dm:252]: Non capitalised

File: code/modules/mob/new_player/new_player.dm [hunk 3]
```diff
@@ -261,6 +261,7 @@
 	if(!is_job_whitelisted(src, rank))	 return 0
 	if(!job.player_old_enough(client))	return 0
 	if(job.admin_only && !(check_rights(R_EVENT, 0))) return 0
+	if(job.mentor_only && !(check_rights(R_ADMIN|R_MENTOR, FALSE))) return FALSE
```
Comment (farie82) [inline code/modules/mob/new_player/new_player.dm:264]: Styling here is conflicting with the rest of the codebase. The checks above do as well though

File: code/game/objects/items/devices/radio/headset.dm (hunk 2 above)
Comment (ZomgPonies) [reply code/game/objects/items/devices/radio/headset.dm:252]: Yeah I actually fixed the ark soft rep's capitalization instead -- no headsets are capitalized in the codebase.

File: code/modules/mob/new_player/new_player.dm (hunk 3 above)
Comment (ZomgPonies) [reply code/modules/mob/new_player/new_player.dm:264]: Yeah I'd rather not make minor styling fixes here that upstream hasn't fixed, it'll just create useless conflicts down the line.

**Pull request merged**